### PR TITLE
change default surface scheme to nothing

### PR DESCRIPTION
**Pull request by szy21**
This PR changes the default surface scheme to nothing. So we will need to set e.g. `--surface_scheme bulk` in the command line argument if we want surface flux in the future. cc @jiahe23

### CODE REVIEW

**Comment by szy21** [comment]
bors try

**Comment by szy21** [comment]
@charleskawczynski What did I do wrong for the doc build?

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -239,31 +239,23 @@ steps:
         artifact_paths: "sphere_held_suarez_rhoe_int/*"
 
       - label: ":computer: held suarez (ρe_tot) equilmoist"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200secs --t_end 3days"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true  --surface_scheme bulk --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200secs --t_end 3days"
```
Comment (charleskawczynski) [inline .buildkite/pipeline.yml:242]:
```suggestion
        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --surface_scheme bulk --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200secs --t_end 3days"
```

Comment (charleskawczynski) [reply .buildkite/pipeline.yml:242]: I think my docs script needs to be slightly more robust

Comment (szy21) [reply .buildkite/pipeline.yml:242]: Ah, good catch, thanks!

**Pull request merged**